libraries/bezier-rs/src/subpath/solvers.rs

```diff
@@ -59,6 +59,62 @@ impl Subpath {
 
 		intersection_t_values
 	}
+
+	/// Return the min and max corners that represent the bounding box of the subpath.
+	pub fn bounding_box(&self) -> Option<[DVec2; 2]> {
+		let bounding_boxes = self.iter().map(|bezier| bezier.bounding_box()).collect::<Vec<[DVec2; 2]>>();
+		if bounding_boxes.is_empty() {
+			// There is no bounding box for empty subpath.
+			None
```

Comment:
@0HyperCube do you think this should return `None` in this case, or `[(0, 0), (0, 0)]`?

Reply:
It doesn't really matter, I personally prefer `None` as it makes it clear to users of this function that there may not always be a bounding box (they can call `.unwrap_or_default()` if they want your functionality)

Reply:
Sounds good, thanks for your insight.